### ISSUE TITLE
Maintainers or Abandoned?

### ISSUE THREAD

**Issue by tm1000** [webextension-toolbox/webextension-toolbox#569]
@balcsida I'm looking to see what the status of this repository is and the organization itself. It seems like you are the only member. There are some pretty big features/bugs coming out that the community has already provided fixes for that I think should have priority in releases but it's not clear where the direction of this project is going.

Example(s):
- #196 
- #373 
- #143

Do you need more collaborators? Or maintainers? Open Source burnout is a real thing and this extension is very nifty. I am willing to help in that way if you'd like.

**Comment by pbek** [comment]
In about 11 month it's game over for Chrome extensions. 😢 

January 2023: The Chrome browser will no longer run Manifest V2 extensions.
See: https://developer.chrome.com/docs/extensions/mv3/mv2-sunset/

**Comment by tm1000** [comment]
@pbek Yes it's too bad. I emailed the maintainer and he got back to me a few days later and said he had to talk to someone else. That person never got back. This project is dead. No one should use it. It should be forked

**Comment by balcsida** [comment]
Hey folks,

So sorry for not getting back to you after @HaNdTriX was not responsive.
I sent you an invite to the project, @tm1000

**Comment by tm1000** [comment]
@balcsida thanks. I accepted your invite. 

Since you have no interest in this and also @HaNdTriX isn't responding can you please also make me an owner. As if you disappear then I will be in a sore position to maintain this

Additionally who owns the npm login

**Comment by balcsida** [comment]
To be honest, you are completely right @tm1000, I feel burned out. This project was fun while there was a project that I could use it on. I no longer have the time or energy to develop WebExtensions and that caused the obvious downfall of this project.

Tried to keep it up, and all I got in return was DMs and emails for free help.
Hats off to you folks, I don't know how you can work on Open Source stuff after an exhausting workday/week.

My deepest apologies for not being responsive and replying to you or anyone else here.

**Comment by balcsida** [comment]
@tm1000 yup, modified you to be owner (forgot to select the correct one on invitation).
Is your NPM username the same as your GitHub user?

**Comment by tm1000** [comment]
Yes my npm is the same
